public/src/components/calls/pre_call_btn_grp.jsx

```diff
@@ -0,0 +1,62 @@
+import React, { Component } from 'react';
+import { ButtonGroup, Button } from 'react-bootstrap';
+
+export default class PreCallButtonGroup extends Component {
+  constructor(props) {
+    super(props);
+    const handlers = ['handleStartCallingClick', 'handleStopCallingClick', 'handleSkipClick', 'handleBadNameClick'];
+    handlers.forEach((handler) => {
+      this[handler] = this[handler].bind(this);
+    });
+  }
+
+  handleStartCallingClick() {
+    const { updateCallStatus } = this.props;
+    updateCallStatus('IN_PROGRESS');
+  }
+
+  handleSkipClick() {
+    // const { call_id } = this.props;
+    console.log('skip requested', this.props);
+  }
+
+  handleBadNameClick() {
+    // const { call_id } = this.props;
+    // send update for bad number(marking call as assigned)
+    // will also need to assign a new call
```

Comment:
No, this follows the regular call flow after submit. No need to short circuit and assign a new call.

Reply:
atm badnameclick just sends an action that updates and gives the user a new call, this is old code.